tests/ctst/world/Zenko.ts

```diff
@@ -424,35 +508,44 @@ export default class Zenko extends World {
         }
 
         //assign the assumed session credentials to the Assumed session.
-        this.parameters.AssumedSession = extractPropertyFromResults(assumeRoleRes, "Credentials");
+        const res = extractPropertyFromResults(assumeRoleRes, 'Credentials');
+        // necessary to avoid linter error
+        if (res) {
+            this.parameters.AssumedSession = res;
+        } else {
+            throw new Error(`Error when trying to assume role ${roleArnToAssume} as service user ${serviceUserName}.
+            No credentials returned.`);
+        }
         this.resumeAssumedRole();
     }
 
     /**
      * Hook Zenko is a utility function to prepare a Zenko
-     * @param {object} parameters - the client-provided parameters
+     * @param {Object.<string,*>} parameters - the client-provided parameters
      * @returns {undefined}
      */
-    static async init(parameters: any) {
+    static async init(parameters: Record<string, unknown>) {
+        CacheHelper.parameters ??= {};
         if (!CacheHelper.accountAccessKeys) {
             CacheHelper.adminClient = await Utils.getAdminCredentials(parameters);
 
             let account = null;
             // Create the account if already exist will not throw any error
             await SuperAdmin.createAccount({
-                name: parameters.AccountName || Constants.ACCOUNT_NAME,
+                accountName: parameters.AccountName as string || Constants.ACCOUNT_NAME,
```

Comment:
to be defined above and then
```suggestion
                accountName: parameters.AccountName || Constants.ACCOUNT_NAME,
```

Reply:
Seems to be same problem as [here](https://github.com/scality/Zenko/pull/1818#discussion_r1179232321)